### PR TITLE
Update ScannerDB image

### PR DESCRIPTION
**Pull request by RTann**
Install Postgres 12 via instructions inspired by https://www.postgresql.org/download/linux/redhat/.
This removes the need to install individual rpms.

Updated the docker-entrypoint by copying https://github.com/docker-library/postgres/blob/master/12/bullseye/docker-entrypoint.sh

Added `STACKROX MODIFIED` to any comments added by this repo

### CODE REVIEW

File: image/db/rhel/scripts/docker-entrypoint.sh
Comment (c-du) [inline image/db/rhel/scripts/docker-entrypoint.sh:42]: Did you see real error for this?

File: image/db/rhel/Dockerfile
Comment (c-du) [inline image/db/rhel/Dockerfile:33]: Good way to go.

File: image/db/rhel/scripts/docker-entrypoint.sh
Comment (RTann) [reply image/db/rhel/scripts/docker-entrypoint.sh:42]: No, this is just directly lifted from https://github.com/docker-library/postgres/blob/master/12/bullseye/docker-entrypoint.sh

We haven't updated this file in years, so I just copy pasted the latest version here and added comments in sections marked with `STACKROX MODIFIED`

**Review by c-du — APPROVED** [review]

**Comment by RTann** [comment]
Going to merge this after the next Scanner release is cut (which will be soon), so we have plenty of time for this to bake for the next release. Once merged I will make an equivalent PR in the stackrox repo

**Pull request merged**
